fedora-packages.spec

```diff
@@ -53,72 +50,45 @@ Requires: xapian-bindings-python
 Requires: python-dogpile-core > 0.4.0
 Requires: python-dogpile-cache > 0.4.1
 Requires: python-memcached
-Requires: packagedb-cli
+# Requires: packagedb-cli
 Requires: python-markdown
 Requires: pygobject3
 Requires: fedmsg
 Requires: python-pdc-client
 Requires: python-webhelpers
 Requires: libappstream-glib
-# For spectool
-Requires: rpmdevtools
 Requires: python-daemon
 Requires: python-webob
 
-Obsoletes: myfedora
-Conflicts: fedoracommunity
+# For spectool
+Requires: rpmdevtools
 
 
 %description
 Fedora-packages is a web application that allow the user to search for packages inside Fedora.
+
 %prep
-%autosetup -n fedora-packages-%{version}
+%autosetup -n %{oldname}-%{version}
 
 %build
 %py2_build
 
 %install
-%{__rm} -rf %{buildroot}
-%{__python2} setup.py install -O1 --skip-build \
-    --install-data=%{_datadir} --root %{buildroot}
-%{__python2} setup.py archive_fedoracommunity_resources -f -o %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets -d moksha -d fedoracommunity
-
-# Hack, to work around tw2 resource archiving issues
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.jquery_template/static/javascript
-cp fedoracommunity/widgets/static/javascript/jquery.tmpl.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.jquery_template/static/javascript
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.connectors.widgets.widgets/static/js
-cp fedoracommunity/connectors/widgets/static/js/fcomm.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.connectors.widgets.widgets/static/js
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.grid/static/javascript/ui/
-cp fedoracommunity/widgets/static/javascript/ui/moksha.ui.{grid,popup}.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.grid/static/javascript/ui
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.package.updates/static/javascript/
-cp fedoracommunity/widgets/package/static/javascript/bodhi.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.package.updates/static/javascript/
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.expander/static/javascript/
-cp fedoracommunity/widgets/static/javascript/jquery.expander.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.expander/static/javascript/
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.tree/static/javascript/
-cp fedoracommunity/widgets/static/javascript/jquery.jstree.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.tree/static/javascript/
```

Comment:
All the JS is removed?

Reply:
The JS is provided by toscawidget (independent rpms). I am not quite why it was packaged in this rpm in the first place. But the app is using the js coming from toscawidget rpms so we can remove these safely.